tracing-appender/src/builder.rs

```diff
@@ -0,0 +1,88 @@
+use std::sync::atomic::AtomicUsize;
+use time::OffsetDateTime;
+use crate::rolling::{create_writer_file, Inner, RollingFileAppender, Rotation};
+use crate::sync::RwLock;
+
+#[cfg(feature="compression")]
+use crate::compression::CompressionConfig;
+
+use crate::writer::WriterChannel;
+
+pub struct RollingFileAppenderBuilder {
+    log_directory: Option<String>,
+    log_filename_prefix: Option<String>,
+    rotation: Option<Rotation>,
+    next_date: Option<AtomicUsize>,
```

Comment:
why does this have to be provided as a builder parameter?

Reply:
Fixed